webpack.config.js

```diff
@@ -3,7 +3,7 @@
 const
     webpack = require('webpack'),
     path = require('path'),
-    prod = process.env.NODE_ENV === 'production',
+    //prod = process.env.NODE_ENV === 'production',
```

Comment:
It is a webpack 5 requirement

Reply:
Ah, I think you're talking about this:

> webpack 5 does no longer include a polyfill for the Node.js `process` variable. Avoid using it in the frontend code.

This isn't front-end code, so the `process` variable should be defined. Are you getting an error? To be clear, we need the `NODE_ENV` environment variable working in order to consider the PR.

Reply:
Found it 
Level 5: Runtime Errors.

This is tricky. You probably have to debug to find the problem. A general advise is difficult here. But we do list some common advices below regarding Runtime Errors:

process is not defined.
webpack 5 does no longer include a polyfill for this Node.js variable. Avoid using it in the frontend code.
Want to support browser usage? Use the exports or imports package.json field to use different code depending on the environment.
Also use the browser field to support older bundlers.
Alternative: Wrap code blocks with the typeof process checks. Note that this will have a negative impact on the bundle size.
Want to use environment variables with process.env.VARIABLE? You need to use the DefinePlugin or EnvironmentPlugin to define these variables in the configuration.
Consider using VARIABLE instead and make sure to check typeof VARIABLE !== 'undefined' too. process.env is Node.js specific and should be avoided in frontend code.
404 errors pointing to URLs containing auto
Not all ecosystem tooling is ready for the new default automatic publicPath via output.publicPath: "auto"
Use a static output.publicPath: "" instead. 
From https://webpack.js.org/migrate/5/#run-a-single-build-and-follow-advises

Reply:
I now understand